### PR TITLE
API-24: Get a single attribute

### PR DESCRIPTION
**Pull request by momoss**
Add controller, route and tests for getting a single attribute

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | :negative_squared_cross_mark:
| Added Behats                      | :negative_squared_cross_mark:
| Added integration tests           | :white_check_mark:
| Changelog updated                 | :negative_squared_cross_mark:
| Review and 2 GTM                  | :white_check_mark:
| Micro Demo to the PO (Story only) | :negative_squared_cross_mark:
| Migration script                  |:negative_squared_cross_mark:
| Tech Doc                          | :negative_squared_cross_mark:

### CODE REVIEW

File: src/Pim/Bundle/ApiBundle/Controller/Rest/AttributeController.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+{
+    /** @var AttributeRepositoryInterface */
+    protected $repository;
+
+    /** @var NormalizerInterface */
+    protected $normalizer;
+
+    /**
+     * @param AttributeRepositoryInterface $repository
+     * @param NormalizerInterface         $normalizer
```
Comment (anaelChardan) [inline src/Pim/Bundle/ApiBundle/Controller/Rest/AttributeController.php]: Align or not ? :D

File: src/Pim/Bundle/ApiBundle/Controller/Rest/AttributeController.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    /**
+     * @param Request $request
+     * @param string  $code
+     *
+     * @throws NotFoundHttpException
+     *
+     * @return JsonResponse
+     */
+    public function getAction(Request $request, $code)
```
Comment (solivier) [inline src/Pim/Bundle/ApiBundle/Controller/Rest/AttributeController.php]: Maybe I missed something, but we already have an AttributeController that returns an Attribute ?

Comment (solivier) [inline src/Pim/Bundle/ApiBundle/Controller/Rest/AttributeController.php]: And what about the rights ?

Comment (momoss) [inline src/Pim/Bundle/ApiBundle/Controller/Rest/AttributeController.php]: Yes, we have but don't return value normalize to standard format, and the rights question will come soon.

Comment (ahocquard) [inline src/Pim/Bundle/ApiBundle/Controller/Rest/AttributeController.php]: It's ok : no right for the moment.

**Pull request merged**